options.go

```diff
@@ -127,7 +133,9 @@ type Options struct {
 	// DropSpanLogs turns log events on all Spans into no-ops.
 	DropSpanLogs bool `yaml:"drop_span_logs"`
 
-	// Set Verbose to true to enable more text logging.
+	// DEPRECATED: the LS tracer no longer logs to stdout. See the documentation
```

Comment:
Slightly incorrect. "DEPRECATED: The LightStep library prints the first error to stdout by default. To print subsequent messages, register ` lightstep.OnEvent(NewOnEventLogger())`.

Reply:
Good catch